components/builder-depot/doc/api.raml

```diff
@@ -196,9 +196,34 @@ securitySchemes:
                                         "origin_id": "77731431660388352"
                                     }
 /pkgs:
+    /search:
+        /{query}:
+            get:
+                description: Search for packages with a query string
+                queryParameters:
+                    distinct:
+                        description: Whether to show a distinct list of packages or not
+                        type: boolean
+                        required: false
+                        default: false
+                        example: true
+                responses:
+                    200:
+                        description: Packages were found and fit on one page
+                    204:
+                        description: Packages were found and require pagination
```

Comment:
The 204 was a mistake on my part.  The server code specifies `status::PartialContent` which off the top of my head I thought was 204.  It's actually 206.  I'll update the docs to match that.

The endpoint itself on the server side _does_ provide pagination controls.  It's possible that the UI isn't taking advantage of them, but these docs are for the server, not the client.